remotespark/pysparkkernel/kernel.js

```diff
@@ -0,0 +1 @@
+IPython.CodeCell.config_defaults.highlight_modes['magic_text/x-sql'] = {'reg':[/^%%sql/, /^%%hive/]};
```

Comment:
The default here is the ANSI SQL dialect, [there's also a hive mime mode that supports additional keywords with the Hive SQL dialect](https://github.com/codemirror/CodeMirror/blob/c4387d6073b15ccf0f32773eb71a54f3b694f2f0/mode/sql/sql.js#L394-L402).